models/topics.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection.js');
+
+const selectTopics = () => {
+  console.log("In selectTopics() in topics.model!")
+
+  return db.query(`SELECT * FROM topics`)
+    .then((topics) => {
+      if (!topics.rows.length)
+        return Promise.reject({status: 404, msg: 'No topics found!'});
```

Comment:
I don't think there is any way to hit this promise rejection / create a test case for no topic being returned.   So I say this isn't needed, but not a big deal!